build.gradle

```diff
@@ -35,7 +36,10 @@ repositories {
 dependencies {
     implementation "org.reflections:reflections:${reflections}"
     implementation "org.projectlombok:lombok:${lombok}"
+    annotationProcessor "org.projectlombok:lombok:${lombok}"
```

Comment:
Doesn't implementation handle it?

Reply:
I'm not an expert in gradle, but when I remove this, the project is not built